### DISCUSSION TITLE
How to modify version dropdown and show all versions?

### DISCUSSION THREAD

**Discussion by mdobydullah** [thuliteio/doks#1293] [Q&A] [answered]
Hi, could you please help me locate the configuration file to modify the version dropdown?

I also want to display all versions. What is the best way to achieve that?

![Screenshot_5](https://github.com/user-attachments/assets/175f2330-cb04-42d2-85f6-a4bdcc634f8b)

Thank you.

**Comment by h-enk** [comment, marked as answer]
New documentation is in the making. Until then, see the old docs: [Versioned Documentation](https://deploy-preview-329--getdoks.netlify.app/docs/how-to/extra-features/versioned-documentation/)

Note, there are several ways to set up versioned docs — it's up to you to implement a process (like for example the branch deploys setup)

**Reply by mdobydullah** [reply]
Thank you so much.

**Reply by h-enk** [reply]
No problem. Like for [Multilingual mode](https://getdoks.org/docs/guides/i18n/), I will set up an example repo and demo website (next to a guide)

**Comment by safaci2000** [comment]
@h-enk is there at ticket or release to watch out for? I've been meaning to add this feature in but it seems like the Versioning pattern is in transition? I'd love to just upgrade and add the versioning support once it's ready.

**Reply by h-enk** [reply]
It's planned for Doks 1.8 (the upcoming release) — see also [Roadmap 2025](https://github.com/orgs/thuliteio/discussions/462)

I can't give you timelines though, but I'm working on it
